doc/files.md

```diff
@@ -26,40 +26,42 @@ Linux    | `$HOME/.bitcoin/`
 macOS    | `$HOME/Library/Application Support/Bitcoin/`
 Windows  | `%APPDATA%\Bitcoin\` <sup>[\[1\]](#note1)</sup>
 
-2. The non-default data directory path can be specified by `-datadir` option.
+2. The data directory path can be specified with `-datadir` option.
```

Comment:
s/with/with the/

s/option/configuration option/

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/20076#issuecomment-703294876).